onnxruntime.sh

```diff
@@ -1,6 +1,6 @@
-package: onnxruntime
+package: ONNXRuntime
 version: "1.7.2"
-tag: c073c88ef
+tag: 4531d55f6
 source: https://github.com/saganatt/onnxruntime.git
```

Comment:
Please let's move this to an official alisw repository using the tagging procedures outlined in:

https://github.com/alisw/alidist#guidelines-for-handling-externals-sources

Can you provide me the original github repo to fork it in alisw?

Reply:
The original repo is here: https://github.com/microsoft/onnxruntime

Reply:
I based my changes on tag 1.7.2. I can prepare the forked repo myself as well, if you point me to the fork.

Reply:
https://github.com/alisw/onnxruntime/tree/alice/v1.7.2 is the branch you should use.

Reply:
Ok, I need one more advice: how I can contribute? It doesn't allow me to push, neither I can make a PR to it...?

Reply:
You need to fork and create a PR. How does it fail?

Reply:
I pushed now. Turned out I needed to delete my previous fork, and re-fork it.